app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, original_file, new_file = command.split()
+    if command_name != "cp":
+        return
+    if original_file != new_file:
+        with open(original_file, "r") as file_get, \
```

Comment:
U can just write here if command_name == "cp" and original_file != new_file:
No need to use 2 if here